src/harness/testcases/spectrum_inquiry_testcase.py

```diff
@@ -391,20 +391,40 @@ def test_WINNF_FT_S_SIQ_16(self):
       self.assertEqual(resp['response']['responseCode'], 0)
 
   @winnforum_testcase
-  def test_WINFF_FT_S_SIQ_17(self):
+  def test_WINNF_FT_S_SIQ_17(self):
     """This test ensures that when a message with two requests, 
     both for PAL channels, is sent with valid parameters, 
     the SAS sends a successful response for both the requests.
     Response Code must be 0 for all CBSDs
     """
 
-    # Register the devices
+    # Load the Data
+    pal_low_frequency = 3550000000.0
+    pal_high_frequency = 3700000000.0
```

Comment:
according to the spec "WINNF-15-S-011" the pal license may aggregate up to **four** PAL channels, so I think giving15 channels is not coherent with that.
and the PAL Id is by census tract by frequency channel so one pal record for one census tract on one channel.